### ISSUE TITLE
Write guide explaining how to use the local SSH daemon as a Nebula status CLI

### ISSUE THREAD

**Issue by briankelly** [DefinedNet/nebula-docs#12]
stub for `src/data/docs/en/cli.md`

```markdown
---
title: Get Status With CLI
slug: cli
summary: cli
---
# Get Status With CLI
```

**Comment by jasikpark** [comment]
I think this issue is tracking a full documentation of the commands available if you enable `ssh` feature?

https://nebula.defined.net/docs/config/sshd/

**Comment by jasikpark** [comment]
Fixed by #119
